app/scripts/modules/core/src/manifest/stage/JobManifestPodLogs.tsx

```diff
@@ -4,15 +4,15 @@ import * as classNames from 'classnames';
 
 import { InstanceReader, IInstanceConsoleOutput, IInstanceMultiOutputLog } from 'core/instance/InstanceReader';
 
-import { IManifestEvent, IManifest } from 'core/domain/IManifest';
-
-import { get, trim, bindAll } from 'lodash';
+import { bindAll } from 'lodash';
```

Comment:
Ugh sorry I know you didn't create this file but can you please move the lodash import up to the third-part import block with his friends react etc.? :pray: